ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -438,6 +439,22 @@ func (s *Service) EntityType() types.Type {
 	return types.Service
 }
 
+func (s *Service) GetTags() ([]*ecs.Tag, error) {
```

Comment:
Hm, small nit, but I might call this something more like `loadTags` or `populateTags` to disambiguate from `utils.GetTags`.